fail2ban/server/action.py

```diff
@@ -394,20 +394,16 @@ def substituteRecursiveTags(cls, tags):
 					# recursive definitions are bad
 					#logSys.log(5, 'recursion fail tag: %s value: %s' % (tag, value) )
 					return False
-				elif found_tag in cls._escapedTags:
-					# Escaped so won't match
+				if found_tag in cls._escapedTags or not tags.has_key(found_tag):
+					# Escaped or missing tags - just continue on searching after end of match
+					# Missing tags are ok - cInfo can contain aInfo elements like <HOST> and valid shell
+					# constructs like <STDIN>.
+					m = t.search(value, m.end())
```

Comment:
why ```m.end``` and not ```m.start()+1```  - couldn't it be some obscure "embbeded" tag e.g. "<<MATCH_ME>OST>" where we would define ```MATCH_ME``` to be either H or IPV6H and assume that it would work? ;-)  (just hypothesising I guess since we seems to have such unittests)